src/WorkbenchAccessManager.php

```diff
@@ -403,4 +433,39 @@ public function flushFields() {
     drupal_set_message($this->t('Field settings reset.'));
   }
 
+  /**
+   * Loads the saved role sections for a given role ID.
+   *
+   * @param string $role_id
+   *   The role ID.
+   *
+   * @return array
+   *   Sections for role.
+   */
+  protected function loadRoleSections($role_id) {
```

Comment:
With this refactoring, it's pretty clear that this is in fact another service (in my opinion)